docs/usage/docker.md

```diff
@@ -141,20 +148,20 @@ Add all paths to ignore into the `ignorePaths` configuration field. e.g.
 }
 ```
 
-##### Enable Docker major updates
+### Enable Docker major updates
 
 Add `"docker:enableMajor"` to your `extends` array.
 
-##### Disable digest pinning
+### Disable digest pinning
 
 Add `"default:pinDigestsDisabled"` to your `extends` array.
 
-##### Automerge digest updates
+### Automerge digest updates
 
 Add `"default:automergeDigest"` to your `extends` array.
 Also add `"default:automergeBranchPush"` if you wish for these to be committed directly to your base branch without raising a PR first.
 
-##### Registry authentication
+### Registry authentication
```

Comment:
By the way, it seems that we are recommending to our readers to put their DockerHub username/password in plaintext in the Renovate config file?

Isn't there a safer way to do this? It seems a really bad idea for users to follow this example? Am I missing something here?

At the very least shouldn't we be warning users to be very careful with this?

Reply:
We could give an example of `password: process.env.DOCKER_HUB_PASSWORD,`

Reply:
Changed in new commits. 😄